misc/gcd.cpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file gcd.cpp
+ * 
+ * @brief 
+ * The idea is, GCD of two numbers doesn’t change if smaller number is subtracted from a bigger number. 
+ * More about  Euclidean algorithm-> https://www.geeksforgeeks.org/euclidean-algorithms-basic-and-extended/
+ * @details 
+ * Time Complexity: O(min(a,b))
+ * Auxiliary Space: O(min(a,b))
+ * @author Akash (https://github.com/aka-0803)
+ * @copyright Copyright (c) 2022
+ * 
+ */
+
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int gcd(int a, int b)

```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.